### DISCUSSION TITLE
How to get coordinates from geocodes

### DISCUSSION THREAD

**Discussion by Lineflyer** [cgeo/cgeo#10549] [Q&A] [answered]
_Originally posted by @Le-Juan in https://github.com/cgeo/cgeo/discussions/9516#discussioncomment-702913_

Posted as a new discussion for following up (seems creating discussions is limited to collaborators, while all users can reply).

Hi!

I think discussions would be a good thing, maybe even something like a discord server might work. ;)
However I wanted to start a QA thread, but I saw that I wasn't able to so, I'll introduce myself here.

I'm Jan-Willem, you can call me Juan, I'm a bigtime user of C:Geo and it inspired me to start a personal project. My personal project is quite similar to GeoKrety and I was hoping to find some answers here about how to connect to geocaching, or at least be able to couple the geocachecode to coordinates.

**Comment by SammysHP** [comment, marked as answer]
The only answer we are allowed to give you is: Groundspeak provides an API that you can use to access their data.

I'm pretty sure your use case is not allowed by the terms in the license agreement for multiple reasons like the competition for their own commercial travelbug system, using data not for users but for another service, requests not linked to a user etc.

I highly doubt that you will get your data in a legal (not prohibited by the terms of service) way.

**Reply by Le-Juan** [reply]
Welp... RIP my project 😂, but at least thanks for the answer... I'll try to think of other ways then to be able to couple the data.

**Comment by eddiemuc** [comment]
@SammysHP just for my understanding: if such a use case is not allowed, how is e.g. GeoKrety handling this?

**Reply by Lineflyer** [reply]
AFAIK they built an own database based on users inputs, i.e. I can enter a geocode and in case its not found in their DB Geokrety asks to provide the coordinates for this log.  @kumy Correct me, if I am wrong.

**Reply by kumy** [reply]
Exactly! Thanks for answering.

(Got the exact same question on GK side https://github.com/geokrety/geokrety-website/discussions/649)

https://geokrety.org/help.php#locationdlagc
Filips wrote in the help page some year ago ;

> Note: according to groundspeak, geocaching.com users may use and log geokrety ;):
> I have checked with Brian, and his definitive answer is that these 'rogue' TBs are not of concern to us - reviewers and GS staff. We cannot govern their usage, and Geocachers are allowed to give out GC codes.
